ci/_init.sh

```diff
@@ -31,13 +31,14 @@ if [[ -z "$TARGET_PLATFORM" ]]; then
   fi
 fi
 if [[ "$TARGET_PLATFORM" == "arm64" ]]; then
-  export DOCKER_MARK="ubuntu20-aarch64"
+  export DOCKER_MARK="rhel8-aarch64"
   declare -A BUILD_IMAGE_NAMES=(
-    [$DRIVER_NAME-$DOCKER_MARK]=$DOCKER_REGISTRY_NAME/client-$DRIVER_NAME-ubuntu20-aarch64:$BUILD_IMAGE_VERSION_AARCH64
+    [$DRIVER_NAME-$DOCKER_MARK]=$DOCKER_REGISTRY_NAME/client-$DRIVER_NAME-$DOCKER_MARK:$BUILD_IMAGE_VERSION_AARCH64
   )
 
   declare -A TEST_IMAGE_NAMES=(
-    [$DRIVER_NAME-$DOCKER_MARK]=$DOCKER_REGISTRY_NAME/client-$DRIVER_NAME-ubuntu20-aarch64:$TEST_IMAGE_VERSION_AARCH64
+    [$DRIVER_NAME-$DOCKER_MARK]=$DOCKER_REGISTRY_NAME/client-$DRIVER_NAME-$DOCKER_MARK:$TEST_IMAGE_VERSION_AARCH64
+    [$DRIVER_NAME-$DOCKER_MARK]=$DOCKER_REGISTRY_NAME/client-$DRIVER_NAME-ubuntu20-aarch64:3
```

Comment:
can we have here a property for the image version or we are going to drop it eventually?

Reply:
done